Zend/zend_builtin_functions.stub.php

```diff
@@ -44,6 +44,10 @@ function is_subclass_of(mixed $object_or_class, string $class, bool $allow_strin
 /** @param object|string $object_or_class */
 function is_a(mixed $object_or_class, string $class, bool $allow_string = false): bool {}
 
+/**
+ * @return array<string, mixed|ref>
```

Comment:
Explicit type is unnecessary (also below)

Reply:
Why is it unnecessary? We don't really care about the `int` key type?

Reply:
Uh yeah, you're right.